### PR TITLE
Implements !verify

### PR DESCRIPTION
**Pull request by dlzou**

### CODE REVIEW

File: apps/discordbot/bot.py [hunk 1]
```diff
@@ -171,18 +182,10 @@ def check_thumb(react, _):
         await self.test_channel.send(f"{member} read rules")
 
         await member.send(
-            "Our mail servers are down right now, so unfortunately we can't verify you yet. Stay "
-            "tuned for updates in the #announcements channel!"
+            "Please verify your berkeley.edu email to gain access to the server! "
```
Comment (royh02) [inline apps/discordbot/bot.py:185]: Specify that the user must send a DM to Philbot. There may be confusion with where to send `!verify`

**Comment by royh02** [comment]
I'm curious. Can we potentially make use of [`discord.ext.commands.Bot`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#bots) to clean up our current commands? It definitely beats a series of `on_message` checks and provides a default help command that documents the behaviours of each command better.

**Comment by dlzou** [comment]
@royh02 Interesting, looks like it descends from `Client` so it might be straightforward to switch. I'd like to know if there's a way to limit some commands to certain channels/DM. Would you be down to work on this at some point? That being said, I would still prefer to get this PR merged first (if you don't see any errors) so we can get back to verifying Discord users ASAP.

File: apps/discordbot/bot.py (hunk 1 above)
Comment (dlzou) [reply apps/discordbot/bot.py]: Oops, just saw this. Changed in new commit.

**Review by royh02 — APPROVED** [review]
If the code works with a test server, you can feel free to merge.

**Comment by dlzou** [comment]
Tested in the CSUA-Test server. Just trying to set a good precedent.

**Pull request merged**